src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -1,18 +1,18 @@
 <template>
   <div v-if="isAlterationView" id="business-type">
-    <v-layout>
+    <v-row no-gutters>
       <!-- Row Title -->
-      <v-flex xs3>
+      <v-col cols="3">
```

Comment:
Columns are:
3 for label
7 for content
2 for buttons

Or:
3 for label
9 for edit block

Reply:
Standardize 👍 
It's official now! 😁

Reply:
By the way, I _think_ `xs` is equivalent just specifying the `cols` (ie, the # cols across all scales xs and larger).

If needed, we can tweak larger scales using `sm/md/lg/xl`... https://vuetifyjs.com/en/api/v-col/#props